### PR TITLE
Complete overhaul of app's action buttons

### LINKED ISSUE #4852: Move action buttons to a fixed-size bottom area (not floating on top of text)
We want to have the action buttons on a clear white section at the bottom of screens and not floating on top of text.

There are several places these changes need to happen.

cc @juanmnl

### LINKED ISSUE #4980: Mobile, sidepanel cannot be closed on entity view
**Describe the bug**
On a mobile screen, when viewing an entity that has no main document, the side panel cannot be closed.

**To Reproduce**
1. Open UWAZI on a mobile screen.
2. Create an entity with just a title.
3. View the entity.

**Expected behavior**
The side panel should be closed by default on mobile, and always have a close button.

**Screenshots**
image

**Device (please select all that apply)**
Mobile

**Browser**
Firefox

**Additional context**
Is the screenshot there's also an issue with map controls.  :rotating_light:

### PR DESCRIPTION
**Pull request by RafaPolit**
fixes #4852
fixes #4980

PR checklist:
- [ ] Update READ.me ?
- [ ] Update API documentation ?

QA checklist:
- [ ] Smoke test the functionality described in the issue
- [ ] Test for side effects
- [ ] UI responsiveness
- [ ] Cross browser testing
- [ ] Code review

### CODE REVIEW

**Comment by RafaPolit** [comment]
ToDo:

- [x] Scroll to the bottom of certain Settings screen when adding groups or links that may be added on non-visible places
- [x] Solve infinite scroll on table view if the container is reduced in height (currently container remains large, and last row hidden behind the footer)
- [x] e2e selectors and overhaul checks
- [x] Ensure new functionality (like passing classnames to some components) is tested

File: app/react/ConnectionsList/components/ResetSearch.js [hunk 1]
```diff
@@ -4,19 +4,21 @@ import { connect } from 'react-redux';
...
     const { connectionsGroups } = this.props;
     return (
       <ShowIf if={Boolean(connectionsGroups.size)}>
-        <button onClick={this.props.resetSearch} className="btn btn-primary">
-          <Icon icon="sync" />
-          <span className="btn-label">{t('System', 'Reset')}</span>
+        <button type="button" onClick={this.props.resetSearch} className="btn btn-default">
+          <Icon icon="times" />
+          <span className="btn-label">
+            <Translate>Clear Filters</Translate>
```
Comment (Zasa-san) [inline app/react/ConnectionsList/components/ResetSearch.js:20]: Should there be a `translationKey` specified in the props of the `<Translate />`?

File: app/react/I18N/components/EditTranslationForm.js [hunk 2]
```diff
@@ -133,12 +132,22 @@ export class EditTranslationForm extends Component {
...
-              <span className="btn-label">{t('System', 'Save')}</span>
-            </button>
+            <div className="btn-cluster">
+              <BackButton to="/settings/translations" className="btn-plain" />
+              {importButton}
+            </div>
+            <div className="btn-cluster content-right">
+              <I18NLink to="/settings/translations" className="btn btn-extra-padding btn-default">
+                <span className="btn-label">
+                  <Translate>Cancel</Translate>
```
Comment (Zasa-san) [inline app/react/I18N/components/EditTranslationForm.js:142]: Same as above, I don't recall if these translate components will work with existing translations withouth the prop `translationKey`.

**Review by Zasa-san — APPROVED** [review]
LGTM!

**Pull request merged**